client/src/test/java/com/alibaba/nacos/client/StringUtilsTest.java

```diff
@@ -0,0 +1,58 @@
+package com.alibaba.nacos.client;
+
+import com.alibaba.nacos.client.utils.StringUtils;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import static org.junit.Assert.*;
+import static com.alibaba.nacos.client.utils.StringUtils.*;
+
+public class StringUtilsTest {
+
+    @Test
+    public void testisNotBlank() {
+        assertTrue(isNotBlank("foo"));
+
+        assertFalse(isNotBlank(" "));
+        assertFalse(isNotBlank(null));
+    }
+
+    @Test
+    public void testIsNotEmpty() {
+        assertFalse(isNotEmpty(""));
+
+        assertTrue(isNotEmpty("foo"));
+    }
+
+    @Test
+    public void testDefaultIfEmpty() {
+        assertEquals("foo", defaultIfEmpty("", "foo"));
+        assertEquals("bar", defaultIfEmpty("bar", "foo"));
+    }
+
+    @Test
+    public void testEquals() {
+        assertTrue(StringUtils.equals("foo", "foo"));
+        assertFalse(StringUtils.equals("bar", "foo"));
+    }
+
+    @Test
+    public void testSubstringBetween() {
+        assertNull(substringBetween(null, null, null));
+
+        assertEquals(null, substringBetween("", "foo", ""));
```

Comment:
You could replace this with `assertNull` again.

Reply:
Missed that. Thanks I changed it